### DISCUSSION TITLE
Camera

### DISCUSSION THREAD

**Discussion by Peio47** [robot-soccer-kit/robot-soccer-kit#49] [Q&A] [answered]
Bonsoir,
Petit (voire gros souci) de caméra.
Je travail sous Linux Mint (de mémoire kernel 5.13)
L'application n'affiche pas la webcam. Au contraire c'est la webcam intégrée qui s'allume, sans image dans l'application.
La webcam est bien installée car j'accède à son flux avec VLC.
Par contre sur l'onglet Vidéo, je n'accède pas à la page de choix de la caméra quand j'appuie sur le bouton Stop

J'ai d'ailleurs obtenu plusieurs cette erreur :
https://gitlab.com/Peio47/robot-soccer-kit/-/raw/main/Capture_d_%C3%A9cran_2022-03-21_16-45-40.png

En vous remerciant,
Peio

**Comment by cgaspard3333** [comment]
Bonjour,
Si vous stoppez la capture en appuyant sur le bouton "Stop" et que sur le menu qui s'affiche vous faites "Refresh Camera" avez vous bien 2 Camera ? (Normalement Camera 0, Camera 1). Camera 0 est censée être la Webcam intégrée et Camera 1 la caméra branchée en USB. Sauf autre problème logiciel, il suffit normalement de sélectionner "Camera 1" et de cliquer sur "Start".

Quant à l'erreur qui est affichée et que vous avez mis en pièce-jointe, c'est juste que la case "Rescale" est vide, assurez vous d'avoir mis une valeur à l'intérieur avant de continuer (entre 70 et 100 par exemple). Je rappelle que la valeur de rescale est une diminution du nombre de pixels traités par le logiciel allant de 0% des pixels traités à 100% des pixels traités.

**Comment by Peio47** [comment]
Bonjour,
Malheureusement je n'ai pas toujours accès à cet page. 

Souvent, rien ne se passe quand je clique sur le bouton Stop

**Reply by cgaspard3333** [reply]
Je suis disponible pour faire un test avec TeamViewer quand vous avez un peu de temps.

**Comment by Peio47** [comment]
Je n'aurais pas le temps aujourd'hui car toute la journée au lycée.
Par contre on peut regarder mercredi ou jeudi après-midi.

**Comment by Peio47** [comment]
Bonjour,
Réessayer aujourd'hui.
J'ai fait en sorte que la caméra soit désignée par /dev/video0
Sur l'application j'arrive à la sélectionner, mais rien ne s'affiche sur la page

**Comment by Peio47** [comment, marked as answer]
Rebonjour,
Le champ rescale était vide et empêchait le bon fonctionnement de la vidéo.
J'ai renommé la caméra /dev/video0 et l'application arrive à l'afficher. Par contre le fps atteint péniblement 3

**Reply by cgaspard3333** [reply]
Je me suis douté, c'est ce que j'avais essayé d'expliquer dans mon message précédent. 
Pour les FPS, essayez de baisser la résolution de la caméra et de jouer sur le "rescale"
